apps/web/app/(use-page-wrapper)/settings/(settings-layout)/organizations/roles/_components/usePermissions.ts

```
@@ -82,8 +89,10 @@ export function usePermissions(): UsePermissionsReturn {
           newPermissions.push(`${resource}.${CrudAction.Read}`);
           break;
         case "all":
-          // Add all permissions for this resource
-          const allResourcePerms = Object.keys(resourceConfig).map((action) => `${resource}.${action}`);
+          // Add all permissions for this resource (excluding internal keys)
+          const allResourcePerms = Object.keys(resourceConfig)
+            .filter((action) => !action.startsWith("_"))
+            .map((action) => `${resource}.${action}`);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)